### PR TITLE
Official Check In

### PR DESCRIPTION
**Pull request by felixatdlr**
## Description

Final check-in of the emerging technologies repository.

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ X ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ X ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by SebastianBezold** [comment]
Project contribution issue opened: https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/15736

File: NOTICE.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+CC-BY-4.0:
+The materials in this repository are made available under the terms of the Creative Commons Attribution 4.0 International License, which is available at https://spdx.org/licenses/CC-BY-4.0.html.
+SPDX-License-Identifier: CC-BY-4.0
+
+## Source Code
+
+The project maintains the following source code repositories:
+* https://github.com/eclipse/tractusx
+* https://github.com/eclipse-tractusx/sig-infra
```
Comment (SebastianBezold) [inline NOTICE.md]: This should point to this repo instead of `sig-infra`

File: README.md
Comment (SebastianBezold) [inline README.md:1]: I would at least mention, that this repository is hosting a collection of topics that have been worked on by the "emerging technologies" team during consortia phase and link to the `README.md` files in the sub-directories

**Review by SebastianBezold — CHANGES_REQUESTED** [review]
Hi @felixatdlr,

there is a small adoption to the `NOTICE.md` that you should do.
Please also consider some top level `README.md` content.
The project contribution IP issue is already closed in the meantime

File: README.md
Comment (felixatdlr) [reply README.md:1]: Done, see https://github.com/eclipse-tractusx/emergingtechnologies/pull/6

File: NOTICE.md (hunk 1 above)
Comment (felixatdlr) [reply NOTICE.md]: Updated, see https://github.com/eclipse-tractusx/emergingtechnologies/pull/5/commits/b18d4e6ceb40360da5986b7f9f0e9e71f553abdc

**Review by SebastianBezold — APPROVED** [review]

**Pull request merged**